security/authorization/compiler/main/grpc_authz_policy_compiler.go

```diff
@@ -0,0 +1,33 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	compiler "google.golang.org/grpc/security/authorization/compiler/api"
+)
+
+func main() {
+	inputFile := os.Args[1]
+	outputFile := os.Args[2]
+	compiler.Compile(inputFile, outputFile)
```

Comment:
Users may type in some values you are not expecting(e.g. a non-existing file path), and in my opinion, a good CLI program would be able to instruct users how to enter the correct input. Can you put some thoughts into this direction and see if there are anything we can improve?

Reply:
So I added a quick user input interface. Where if the user does not know how to run the compiler they can run it without any arguments and the program will prompt the user for the correct inputs. Otherwise they can use the compiler directly